ekip/nationalparks/management/commands/field_trip.py

```diff
@@ -5,7 +5,11 @@
 from nationalparks.management.commands import ficorcleaner as fc
 from nationalparks.models import FieldTripSite, BestVisitTime, YouthFacility
 
-
+#Begin by cleaning house!
+FieldTripSite.objects.all().delete()
```

Comment:
You'll need to add a conditional checking if these entities exist. This is causing the build to fail on this branch. @trevorcolenps